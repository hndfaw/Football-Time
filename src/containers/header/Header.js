import React, { Component } from 'react';
import { connect } from 'react-redux';
import logo from '../../images/football-time-logo-4.png';
import './header.css';
// import { setTodaysMatches } from '../../actions';
import { NavLink } from 'react-router-dom';


class Header extends Component {
  
 
  
  

  render() {
    

    return (
      <header>
        <img className="logo" src={logo} alt="football time logo" />
        <section className="header-tabs-container">
            <NavLink to="/">
              All Leagues
            </NavLink>
            <NavLink to="/league">
              One League
            </NavLink>
            
        </section>
      </header>
    )
  }
}

const mapStateToProps = state => ({
  leaguesData: state.leaguesData,
})



export default connect(mapStateToProps, null)(Header)
