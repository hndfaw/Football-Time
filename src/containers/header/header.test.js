import React from 'react';
import { Header } from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {
  it('should match the snopshot', () => {
    let wrapper = shallow(<Header selectedLeague="league524" />);
    expect(wrapper).toMatchSnapshot()
  })
})