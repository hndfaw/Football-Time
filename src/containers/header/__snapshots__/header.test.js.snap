// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snopshot 1`] = `
<header>
  <img
    alt="football time logo"
    className="logo"
    src="football-time-logo.png"
  />
  <section
    className="header-tabs-container"
  >
    <div
      className="nav-btn-container"
    >
      <NavLink
        className="all-leagues-tab"
        to="/todaysmatches"
      >
        Today's Matches
      </NavLink>
      <NavLink
        className="one-league-tab"
        to="/leagues/524"
      >
        Leagues
      </NavLink>
    </div>
  </section>
</header>
`;
